README.md

```diff
@@ -138,6 +137,45 @@ Below is a typical example of how this use case would take place:
 
 - Compute the new root for the vault’s rewards distributor, submit it, wait for the timelock (if any), accept the root, and let vault depositors claim their rewards according to the vault manager’s rewards re-distribution strategy.
 
+## Emergency
+
+### An enabled market is now considered unsafe
+
+If an enabled market is considered unsafe (e.g., risk too high), the curator/owner may want to disable this market in the following way:
+
+- Revoke the pending cap of the market with the `revokePendingCap` function (this can also be done by the guardian).
+- Set the cap of the market to 0 with the `submitCap` function.
+- Withdraw all the supply of this market with the `reallocate` function.
+  If there is not enough liquidity on the market, remove the maximum available liquidity with the `reallocate` function, then put the market at the beginning of the withdraw queue (with the `updateWithdrawQueue` function).
+- Once all the supply has been removed from the market, the market can be removed from the withdraw queue with the `updateWithdrawQueue` function.
+
+### An enabled market reverts
+
+If an enabled market starts reverting, many of the vault functions would revert as well (because of the call to `totalAssets`). To turn the vault back to an operating state, the market must be forced removed by the owner/curator, who should follow these steps :
```

Comment:
```suggestion
If an enabled market starts reverting, many of the vault functions would revert as well (because of the call to `totalAssets`). To turn the vault back to an operating state, the market must be forced removed by the owner/curator, who should follow these steps:
```